### ISSUE TITLE
Contact cypriot theater and tv series writers to get data

### ISSUE THREAD

**Issue by nsorros** [MantisAI/cypriot_bert#27]

**Comment by pedroandreou** [comment]
### **_Channels:_**
- ant1 :x:
- rik :open_hands:
- sigma :open_hands:

### **_:performing_arts: Theatres:_**
- Theatre of Dionysus :open_hands:
- Pantheon Theatre :open_hands:
- Pattihio :open_hands:
- Theatre of Thoc :open_hands:
- Theatre of Nicosia :open_hands:

### **_:people_holding_hands:People:_**
- Tonia Mishiali (Film Director) :heavy_check_mark:
-  Mitko Kiprianidis(Film Director) :heavy_check_mark:
- Naya Director (Director at Alpha Cyprus TV) :heavy_check_mark:
- Petros Charalambous (Film Director) :open_hands:
- Savvas Christou (Film Director) :open_hands:
- Alexander Machlouzarides-Shalit (Film Director) :open_hands:
- Michalis Terlikkas (Cypriot couplets performer) :open_hands:
- Andreas Kyriacou (not contacted yet, [LinkedIn](https://www.linkedin.com/in/thesplashcy/?originalSubdomain=cy))
- Kyriacos Tofarides (not contacted yet, [LinkedIn](https://cy.linkedin.com/in/kyriacos-tofarides-5a07633b))
- Panicos Chrysanthou (not contacted yet, [LinkedIn](https://cy.linkedin.com/in/panicos-chrysanthou-479413a8))

### **_:film_projector: Cypriot Writers that are a must to contact:_**
- Loris Loizides :open_hands:
- Giorgos Tsiakkas (no :information_source: found)
- Hristiana Artemiou (no :information_source: found)
- Corinna Avraamidou (no :information_source: found)
- Annita Nikolaou (no :information_source: found)

**Comment by nsorros** [comment]
I also found this potentially useful resource https://www.cyprusdirectors.com/members

**Comment by pedroandreou** [comment]
> I also found this potentially useful resource https://www.cyprusdirectors.com/members

Indeed, very useful resource; I have already looked into all the directors of this website

**Comment by nsorros** [comment]
> > I also found this potentially useful resource https://www.cyprusdirectors.com/members
> 
> Indeed, very useful resource; I have already looked into all the directors of this website

Do we have their contact numbers or emails? Did we send them an email?

**Comment by pedroandreou** [comment]
> > > I also found this potentially useful resource https://www.cyprusdirectors.com/members
> > 
> > 
> > Indeed, very useful resource; I have already looked into all the directors of this website
> 
> Do we have their contact numbers or emails? Did we send them an email?

I only contacted the ones (see :people_holding_hands: People section above) who:
- could be tracked down
- have shown some demonstration of work online; either that was a Youtube video of their work or their own website/portfolio
- are not directors of short films

**Comment by nsorros** [comment]
Do we have those contacts in our list?
```
donis Florides, Andreas Pantzis, Marinos Kartikkis, Petros Charalambous, Corinna Avraamidou, Kyriakos Toffarides just to name a few. Also, you can find copies of their films at the Cultural Services of Cyprus (Cinema Department). They might not be able to give you copies but you can watch the films in a viewing room. You can contact Diomides Nikitas at 22809811```

**Comment by chr7stos** [comment]
I have contacted Adonis Florides from that list, though he did not have much material

**Comment by pedroandreou** [comment]
> I have contacted Adonis Florides from that list, though he did not have much material

Any chance that one of the materials that he had was the script his film "Rosemarie"?

**Comment by pedroandreou** [comment]
found a new potentially good resource https://film.investcyprus.org.cy/film-cyprus/find-a-professional/
